core/src/main/java/com/sequenceiq/cloudbreak/core/flow2/cluster/datalake/upgrade/validation/event/ClusterUpgradeValidationHandlerSelectors.java

```diff
@@ -4,7 +4,8 @@
 
 public enum ClusterUpgradeValidationHandlerSelectors implements FlowEvent {
 
-    VALIDATE_CLUSTER_EVENT;
+    VALIDATE_CLUSTER_EVENT,
```

Comment:
minor: naming does not reflect that this is used for disk validation, I'd rename it

Reply:
Renamed to VALIDATE_DISK_SPACE_EVENT